src/bin/stratisd.rs

```diff
@@ -75,15 +69,21 @@ fn main() {
         .get_matches();
 
     if matches.is_present("debug") {
-        unsafe { DEBUG = true }
-    };
+        let mut builder = LogBuilder::new();
```

Comment:
I think if you drop the format spec altogether, the format will be the same, at least that's what the code looks like. The global initializer seems to accept the LogBuilder's default format.